src/components/Stations.jsx

```diff
@@ -8,73 +8,70 @@ const useStyles = makeStyles((theme) => ({
     fontFamily: 'Roboto Mono',
     fontStyle: 'normal',
     fontSize: '16px',
-    lineHeight: '16px',
+    lineHeight: '18.4px',
     fontWeight: '500',
   },
 }));
 
-const options = [
-  'ALL LOCATIONS',
-  'ARCHIVE',
-  'AR',
-  'AUS',
-  'CAMS',
-  'EXOSS',
-  'GMN',
-  'MA',
-  'NCA',
-  'TEXAS',
-  'UAE',
-  'BENELUX',
-  'CHILE',
-  'EDMOND',
-  'FL',
-  'LOCAMS',
-  'NAMIBIA',
-  'NZ',
-  'SA',
-  'SONOTACO',
-  'TK',
-];
+const options = {
```

Comment:
Should we use the station abbreviation as the key for this hash map?

Reply:
And then we can use the abbreviation as key for all station options.

Reply:
I have already fixed this, using the full names as the key for the dropdown the abbreviations passed to the request